### DISCUSSION TITLE
faucets

### DISCUSSION THREAD

**Discussion by Milano-xx** [Cyfrin/foundry-full-course-cu#2857] [General]
I've tried all the faucets available but still hasn't gotten any why? somebody help please. They keep asking for ETH in the mainnet

**Comment by EngrPips** [comment]
Hello @Milano-xx, Try this faucet https://sepolia-faucet.pk910.de/

**Reply by Milano-xx** [reply]
it always says i should get enough GIT at least 2

**Reply by EngrPips** [reply]
I will try the faucet and see how it goes

**Reply by onojaaofficial** [reply]
@EngrPips I am currently experiencing the same challenge as @Milano-xx

**Reply by EngrPips** [reply]
Did you tried this https://cloud.google.com/application/web3/faucet/ethereum/sepolia, and it didn't work for you?

**Reply by onojaaofficial** [reply]
> Did you tried this https://cloud.google.com/application/web3/faucet/ethereum/sepolia, and it didn't work for you?

Wow! this worked

**Reply by EngrPips** [reply]
Awesome. I am glad it worked for you.

**Comment by LordAdhaar** [comment]
https://cloud.google.com/application/web3/faucet/ethereum/sepolia

You can use this also

**Reply by Milano-xx** [reply]
but it keeps loading, but never sends anything

**Reply by LordAdhaar** [reply]
Interesting. Maybe try in a bit or check your network connection. Has always worked for me.

**Comment by Milano-xx** [comment]
Yeahh it actually didn't

**Comment by Milano-xx** [comment]
Really how many hours did it take?

**Comment by nixtamer** [comment]
All the faucets i tried ask for at least 0.001 ETH on the ethereum mainnet, anyone experience the same issue?

**Reply by EngrPips** [reply]
You can use this faucet https://sepolia-faucet.pk910.de/

**Reply by nixtamer** [reply]
Thanks, but i've got this message showing up

> 
> New sessions require a minimum gitcoin passport score of at least 2.
> Please visit [Gitcoin Passport](https://passport.gitcoin.co/#/) to increase your passport score.

Any advice on how can i get past that?

**Reply by EngrPips** [reply]
Click on the `Gitcoin Passport` link to link some of your social and your passport score will get increased.

**Reply by nixtamer** [reply]
It din't work for me, anyway [Google cloud faucet](https://cloud.google.com/application/web3/faucet/ethereum/sepolia) did

**Reply by EngrPips** [reply]
Awesome, I am glad something worked at least.
